src/main/webapp/src/app/my_gardens_page/my-gardens.component.css

```diff
@@ -0,0 +1,48 @@
+.avatar {
+  background-size: cover;
+  height: 50px;
+  padding-top: 5px;
+}
+.avatar-image {
+  width: 50px;
+}
+
+.button {
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 0.5em 3em;
+  text-decoration: none;
+  border: none;
+  box-sizing: border-box;
+  box-shadow: inset 0 -0.6em 0 -0.35em #003353;
+  background-color: #005c80;
+  color: white;
+}
+.cards {
+  max-width: 400px;
```

Comment:
nit: make sure all css functions are separated by one line.
@codyjrivera can you please comment the linting command here? I also forget about it

Reply:
Its ok, I know the linting command. When I ran it, it didn't change that.

Reply:
@stephypy `make format` and `make lint` in the top directory